patients/service.go

```diff
@@ -152,7 +188,7 @@ func (s *service) DeletePermission(ctx context.Context, clinicId, userId, permis
 			"deleting patient from clinic because the patient revoked all permissions",
 			"userId", userId, "clinicId", clinicId,
 		)
-		if err := s.Remove(ctx, clinicId, userId); err != nil {
+		if err := s.Remove(ctx, clinicId, userId, &userId); err != nil {
```

Comment:
Is the `userId` the id of the user that actually performed the deletion?

Reply:
In this case we delete the patient because they revoked the permissions, so it makes sense to log that user id